prow/gerrit/adapter/adapter.go

```diff
@@ -223,7 +223,7 @@ func (c *Controller) ProcessChange(instance string, change client.ChangeInfo) er
 		Pulls: []kube.Pull{
 			{
 				Number: change.Number,
-				Author: rev.Commit.Author.Name,
+				Author: rev.Commit.Author.Email,
```

Comment:
Is this breaking?

Reply:
I'm pretty sure the existing author names were only used for display (as they weren't unique)? @krzyzacy

Reply:
we don't really use it for anything besides display the author name on deck (for gerrit) so...

although the mailto link won't be that useful as well - maybe we can query a gerrit user page from author name?

Reply:
> although the mailto link won't be that useful as well - maybe we can query a gerrit user page from author name?

 That makes more sense, will do that :+1: